travis-ci-inspired-dark-extended.user.css

```diff
@@ -2711,7 +2711,7 @@
   }
   @media only screen and (min-width: 50.063em) {
     .tile h2::after {
-      background: linear-gradient(-90deg, #191919 0, rgba(255, 255, 255, 0) 100%) !important;
+      background: none;
```

Comment:
This is not the correct way to address these.

Can you post  screenshots of the pages youre trying to fix and details about the issues?

Also links to the affected pages would be nice too thanks.

Reply:
ping @UTCWebDev